Source/SharpNeedle/SonicTeam/GismoConfig.cs

```diff
@@ -0,0 +1,248 @@
+﻿using SharpNeedle.BINA;
+
+namespace SharpNeedle.SonicTeam;
```

Comment:
Namespace declaration should be at the top.

Reply:
Resolved in 17466e9.